starter-code/views/categories.ejs

```diff
@@ -1,3 +1,18 @@
   <body>
-    <h1>Categories</h1>
+      <h1>Categories</h1>
+
+      <ul>
+          <% categories.forEach((e) => { %>
+            <li>
+              <a href=‘http://localhost:3000/categories?cat=<%= e %>’ >
```

Comment:
OJO CON ESTO!!! la dirección que se pone aquí, no es la completa, es decir, aquí no va `http://localhost:3000/`, si lo hacéis así, la URL en el navegador os queda como `http://localhost:3000/http://localhost:3000/categories?cat=political` y NO FUNCIONA